evgMrmApp/src/evgMrm.cpp

```diff
@@ -92,8 +137,22 @@ evgMrm::evgMrm(const std::string& id,
     m_acTrig(id+":AcTrig", pReg),
   shadowIrqEnable(READ32(m_pReg, IrqEnable))
 {
+    struct VMECSRID info;
+    info.board = 0; info.revision = 0; info.vendor = 0;
+
+    epicsUInt32 slot = busConfig.vme.slot;
```

Comment:
There needs to be a test of `bus_configuration::busType` around the VME specific code so that PCI* EVGs will continue to work.